### ISSUE TITLE
make: *** No rule to make target 'obj/yolt.o', needed by 'darknet'.  Stop

### ISSUE THREAD

**Issue by ABadCandy** [avanetten/yolt#6]
Hi, thank you for your sharing firstly. I have deployed your code in docker but when i try to compile the Makefile it makes mistake like that: 
make: *** No rule to make target 'obj/yolt.o', needed by 'darknet'.  Stop

And I note that the line 24 in Makefile "COMMON=-I/Users/avanetten/anaconda/pkgs/libgdal-2.1.0-0/include" , I just use python2.7 of system and already add PPA for "pip install gdal",  but still make failed.
So how should I fix it?

**Comment by yghlc** [comment]
I ran into the same trouble when I tried to compile the codes. You need to modify the Makefile and corresponding C files. Since that the author upgraded "yolt" to "yolt2". You can try my forked version if you like.
Moreover, the Dockfile did not include "imagemagick" used in the python scripts.

**Comment by ABadCandy** [comment]
@yghlc Yes, I just installed anaconda2 for "conda install gdal" and imagemagick for "convert" command after compiled the Dockerfile. And I found the "pip install tensorflow-gpu"  in Dockfile may install the newest tensorflow which may not be compatible with CUDA8, so I specified the version of keras==2.1.5 and tensorflow-gpu==1.4.0, and the docker works well.  I will try your forked version to correct the Makefile and corresponding C files, thank you!
